### PR TITLE
Update readme.md

### PR DESCRIPTION
**Pull request by DarkStoorM**
`npm i` will now automatically build the source and clone the environment file

* applies Markdown lints

🏃‍♂️ 💨

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
     "test:legacy": "mocha test",
     "start": "node dist/index.js",
     "cloneEnv": "node dist/scripts/cloneEnv.js --clone",
-    "postinstall": "npm run db:migrate",
+    "postinstall": "npm run db:migrate && npm run build:src && npm run cloneEnv",
```
Comment (Hofled) [inline package.json]: Please add `concurrently` to the project, and use that to run multiple npm commands instead of the npm native way. @DarkStoorM

**Review by Hofled — CHANGES_REQUESTED** [review]
Please refer to [this](https://www.npmjs.com/package/concurrently) as a solution.

**Comment by DarkStoorM** [comment]
🎉

**Review by dcepil — APPROVED** [review]
Looking good 🤸

**Comment by Hofled** [comment]
I don't have access to a computer to confirm that it functions properly on a new environment but it indeed looks good 👍
Feel free to merge it when you can

**Review by Hofled — APPROVED** [review]

**Pull request merged**